sui/cli-utils.js

```diff
@@ -51,7 +51,16 @@ const addExtendedOptions = (program, options = {}) => {
     return program;
 };
 
+const addBaseOptionsToCommands = (program, options = {}) => {
+    if (program.commands.length > 0) {
```

Comment:
add to the program itself as well, so you can see these generic options when doing `node ./program --help`

Although, I was wondering if commander could do this for us instead of us manually having to add to each cmd

Reply:
will apply base options to the main program as well. i didn't check the commander lib because it seems easier to just implement it than digging their doc. anyway, will do a bit more read.